ScheduleSample/ScheduleSample/packages.config

```diff
@@ -20,7 +20,7 @@
   <package id="System.IO.Compression" version="4.3.0" targetFramework="portable45-net45+win8" />
   <package id="System.Linq" version="4.3.0" targetFramework="portable45-net45+win8" />
   <package id="System.Linq.Expressions" version="4.3.0" targetFramework="portable45-net45+win8" />
-  <package id="System.Net.Http" version="4.3.1" targetFramework="portable45-net45+win8" />
+  <package id="System.Net.Http" version="4.3.4" targetFramework="portable45-net45+win8" />
```

Comment:
@TamilarasanGunasekaran , In this PR changes showing only one vserion changes - https://github.com/SyncfusionExamples/uwp-import-export-schedule-appointments/pull/4/files

But you have made more changes . Check once.

Reply:
This repo has 3 issues, so I added those fixes in this PR.